packages/api-electron/package.json

```diff
@@ -4,11 +4,16 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "rimraf dist && rollup -c",
+    "test": "npm run build"
```

Comment:
The test script? It's not used by rollup - Travis CI now runs `lerna run test`, which will run `npm test` in all packages, this is just to make sure the build step isn't broken.